about.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://kit.fontawesome.com/adf6e8ca6b.js" crossorigin="anonymous"></script>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" href="images/title-ai.png">
+  <title>About</title>
+</head>
+<body>
+  <header>
+    <nav class="nav1">
+              <a class="nav-link" href="#">Logout</a>
+              <a class="nav-link" href="#">My Page</a>
+              <a class="nav-link" href="#">English</a>
+              <a class="nav-link" href="#" aria-label="twitter"><i id="twitter-icon" class="fa fa-twitter text-white"></i></a>
+              <a class="nav-link" href="#" aria-label="facebook"><i id="facebook-icon" class="fa fa-facebook text-white"></i></a>
+    </nav>
+    <nav class="nav2">
+      <div class="nav2Content">
+        <a id="bordered-link" class="nav-item" href="#">AI Compaign</a>
+        <a class="nav-item" href="#">News</a>
+        <a class="nav-item" href="#">Sponser</a>
+        <a class="nav-item" href="#">Join</a>
+        <a class="nav-item" href="#">Program</a>
+        <a class="nav-item" href="#">About</a>
+      </div>
+      <div class="navLogo"><a href="index.html"><img src="images/Logo.png" alt="logo"></a></div>
+    </nav>
+    <button class="mobile-menu" type="button" ><img src="https://img.icons8.com/ios/50/000000/menu--v4.png" alt="menu"/></button>
+    <div class="mobile-menu-list" id="about-menu">
+      <a class="nav-item, nav-link" href="index.html">Home</a>
+      <a class="nav-item, nav-link" href="#">Program</a>
+      <a class="nav-item, nav-link" href="#">Join</a>
+      <a class="nav-item, nav-link" href="#">Sponser</a>
+      <a class="nav-item, nav-link" href="#">News</a>
+      <a class="nav-item, nav-link" href="#">AI Compaign</a>
+    </div>
+  </header>
+<section class="about">
+  <h1 class="aboutTitle">MACHINE LEARNING<br>ONLONE COURSES 2022</h1>
```

Comment:
- [ ] [OPTIONAL] In the `div` where we have Machine Learning on the about page for mobile, kindly adjust the circular background image as it is currently constricting the Machine learning text. This will enable a good user experience.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/153417060-34e275f9-bd0c-414c-91b6-83d67b065728.png) | ![11](https://user-images.githubusercontent.com/58692590/153417094-5e1c603b-07e6-4933-ab1b-237d11de0342.png)